rgs-hoodies-sql.py

```diff
@@ -59,6 +62,26 @@ def __init__(self, parent, ):
 
         self.tab_switcher(1)
 
+    def create_database(self):
+        conn = sqlite3.connect('hoodies.db')
```

Comment:
The connection can be made once and saved as an attribute of the class to be more efficient